.github/workflows/codeql-analysis.yml

```diff
@@ -9,31 +9,53 @@ on:
 jobs:
   CodeQL-Build:
 
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
No need to mention the version.

Reply:
I think the workflow is not using main.yml

Reply:
It is not using main.yml. It will be better to use the same version of Ubuntu for both workflows. I think fixing the version is better to avoid any surprises when the OS upgrades are available.

Reply:
Updated the ubuntu version